CMakeLists.txt

```diff
@@ -85,6 +89,26 @@ add_subdirectory(source)
 add_subdirectory(examples)
 add_subdirectory(test)
 
+find_program(CLANG_FORMAT NAMES clang-format-10 clang-format-10.0 clang-format)
+
+set(CLANG_FORMAT_REQUIRED "10.0")
+
+if(CLANG_FORMAT)
+	get_program_version_major_minor(${CLANG_FORMAT} CLANG_FORMAT_VERSION)
+	message(STATUS "Found clang-format: ${CLANG_FORMAT} (version: ${CLANG_FORMAT_VERSION})")
```

Comment:
The new code in this file is using tabs but the rest of the project uses 4 spaces for indentation in CMake files.

If we could keep intendation consistent that would be best.

Reply:
Okay. Done.